src/preferences/dialog/dlgprefdeckdlg.ui

```diff
@@ -33,6 +33,16 @@
         </property>
        </widget>
       </item>
+      <item row="4" column="0">
+       <widget class="QLabel" name="labelAutoHotcueColors">
+        <property name="text">
+          <string>Color Hotcues</string>
```

Comment:
The spelling ("hotcue" vs. hot cue") is inconsistent and the case is slightly inconsistent with the remaining labels.

Proposal:
Auto hot cue colors   [ ] Assign predefined colors to newly created hot cue points